application/asset/sass/page-blocks.scss

```diff
@@ -222,5 +223,11 @@
     .site-summary {
         margin: 0 0 0 .5rem;
```

Comment:
Maybe condense the margin styles here. I don't think we even need the left margin on the summary.